_includes/head.html

```diff
@@ -1,39 +1,58 @@
 <head>
-    <meta charset="utf-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <meta name="description" content="Connecting open source & design! 🎨" />
-    <meta name="keywords" content="connecting,open,source,and,design,osd" />
-    <meta name="author" content="" />
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <meta name="description" content="Connecting open source & design! 🎨" />
+  <meta name="keywords" content="connecting,open,source,and,design,osd" />
+  <meta name="author" content="" />
 
-    <title>{% if page.title %}{{page.title}} - {% endif %}Open Source Design</title>
-    <link href="/css/lora.css" rel="stylesheet" type="text/css" />
-    <link href="/css/montserrat.css" rel="stylesheet" type="text/css" />
+  <title>
+    {% if page.title %}{{ page.title }} - {% endif %}Open Source Design
+  </title>
+  <link href="/css/lora.css" rel="stylesheet" type="text/css" />
+  <link href="/css/montserrat.css" rel="stylesheet" type="text/css" />
 
-    <link rel="shortcut icon" href="/images/favicon.ico" />
-    <link rel="apple-touch-icon-precomposed" href="/images/opensourcedesign-logo.png" />
-    <link rel="mask-icon" sizes="any" href="/images/opensourcedesign-logo.svg" color="#000" />
+  <link rel="shortcut icon" href="/images/favicon.ico" />
+  <link
+    rel="apple-touch-icon-precomposed"
+    href="/images/opensourcedesign-logo.png"
+  />
+  <link
+    rel="mask-icon"
+    sizes="any"
+    href="/images/opensourcedesign-logo.svg"
+    color="#000"
+  />
```

Comment:
It looks a bit wrong to put this on separate lines, no? Didn’t do that with the shortcut icon either, and I’ve never seen it like this. :D

Reply:
This is actually intentional, Prettier has a certain maximum line length, and this enhances readability.

Reply:
Ah makes sense, thanks for the explanation! :)